src/main/java/net/dv8tion/jda/api/events/guild/update/GuildUpdateBannerEvent.java

```diff
@@ -55,11 +57,27 @@ public String getNewBannerId()
      * @return The new banner url, or null if the banner was removed
      */
     @Nullable
-    public String getNewBannerIdUrl()
+    public String getNewBannerUrl()
     {
         return next == null ? null : String.format(Guild.BANNER_URL, guild.getId(), next);
     }
 
+    /**
+     * The new banner url
+     *
+     * @return The new banner url, or null if the banner was removed
+     *
+     * @deprecated This will be replaced by {@link #getNewBannerUrl()}
+     */
+    @Nullable
+    @Deprecated
+    @DeprecatedSince("4.1.1")
+    @ReplaceWith("getNewBannerUrl()")
+    public String getNewBannerIdUrl()
+    {
+        return getNewBannerUrl();
```

Comment:
That makes no sense.
If you deprecate the method, then why return the new field?

Reply:
Backwards compatibility.

Reply:
Was on mobile (which sucked a bit) so I saw all of this in a wrong way (assumed getNewBannerUrl() was actually a field xD)
So issue on my end.